src/main/kotlin/no/nav/familie/ef/sak/vilkår/dto/VilkårDto.kt

```diff
@@ -23,7 +23,10 @@ data class VilkårGrunnlagDto(
     val registeropplysningerOpprettetTid: LocalDateTime,
     val adresseopplysninger: AdresseopplysningerDto?,
     val dokumentasjon: DokumentasjonFraSøknadDto?
-)
+) {
+    fun finnesBarnUtenRegistrertForelder() = barnMedSamvær.any { it.registergrunnlag.forelder == null }
+    fun finnesFlereBarnMedSammeAnnenForelder() = barnMedSamvær.groupingBy { it.registergrunnlag.forelder?.fødselsnummer }.eachCount().any { it.value > 1 }
```

Comment:
Tolker nesten dette som at den gir true hvis det finnes en forelder som har fler enn 1 barn
Mens metoden tolker jeg mer som att det finnes flere barn, dvs 
`...eachCount().count { it.value > 1 } > 1`

Burde metoden endres eller burde navnet endres til eks `flereBarnMedSammeForelder`? 

Hvis fødselsnummer på annen forelder mangler på 2 barn, så gir denne true, er det ønskelig? Det kan fortsatt være 2 ulike foreldrer 👀

Reply:
finnesBarnUtenRegistrertForelder - denne vil vel catche om et barn ikke har registrert forelder. Men det kan kanskje hende at det er en annen forelder registrert (altså at annen forelder er not null), og at fødselsnummer ikke finnes?
Det er uansett bra at denne metoden tar utgangspunkt i at ukjente annen foreldre regnes som samme, fordi det fører til at vilkåret _ikke_ blir automatisk vurdert.